OlesiaBarannik/HW5/2.py

```diff
@@ -0,0 +1,27 @@
+# Task2. Print Fibonacci numbers up to the entered number n,
+# using cycles.
+# (Sequence of Fibonacci numbers 0, 1, 1, 2, 3, 5, 8, 13, etc.)
+
```

Comment:
good job
look at this implementation


def logic_of_Fibonacci(user_number):
    list_Fibonacci = [0, 1]
    if user_number == 0:
        return [0]
    elif user_number == 1:
        return list_Fibonacci

    previous_number, second_number = 0, 1
    while True:
        result = previous_number + second_number
        if result > user_number:
            break
        list_Fibonacci.append(result)
        previous_number, second_number = second_number, result

    return list_Fibonacci


user_number = int(input("Please enter a number to get its Fibonacci list\n"))
print(logic_of_Fibonacci(user_number))

Reply:
thank you, I just thought that when 1 should be [0, 1, 1] that's why I used append(1)